circle.yml

```diff
@@ -0,0 +1,29 @@
+machine:
+  python:
+    version: 2.7.12  # this specific version is preinstalled on the CI servers and allows faster builds
+  environment:
+    PYPI_USERNAME: openfisca-bot  # set here the name of your Pypi account to automatically publish your template to Pypi
+    # PYPI_PASSWORD: this value is set in CircleCI's web interface; do not set it here, it is a secret!
+
+dependencies:
+  override:
+    - pip install --upgrade pip wheel  # pip >= 8.0 needed to be compatible with "manylinux" wheels, used by numpy >= 1.11
+    - pip install twine
+    - pip install babel # this line is only needed by internationalization with i18n. We should remove it when we get rid of i18n.
+    - python setup.py compile_catalog  # this line is only needed by internationalization with i18n. We should remove it when we get rid of i18n.
+    - pip install .[test] --upgrade
+test:
+  pre:
+    - git fetch
+  override:
+    - make test
+    - ./check-version-bump.sh
```

Comment:
I thought about it, but I'm not sure it totally works.
What happen if I work on a branch, pass all tests, and _then_ open a PR. Will CircleCI rebuild it with the `CI_PULL_REQUEST` ? Or will it consider that the tests are green, as the commit `sha1` has not changed ?